.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
           ]
           args: ['--max-line-length=88', 
                  '--docstring-convention=numpy', 
-                 '--per-file-ignores=**/tests/**.py:D functional-tests/**:D', 
+                 '--per-file-ignores=**/tests/**.py:D functional-tests/**:D,E501,W291',
```

Comment:
Should this be here or under `--extend-ignore`?

Reply:
This links the ignore to just the functional tests

Reply:
ok